src/memote/__init__.py

```diff
@@ -30,6 +30,18 @@
 from memote.suite.reporting import *
 from memote.jinja2_extension import *
 
+import sys
+import logging
+
 __author__ = "Moritz E. Beber"
 __email__ = "morbeb@biosustain.dtu.dk"
 __version__ = "0.9.12"
+
+LOGGER = logging.getLogger()
```

Comment:
It's still logging within memote so I prefer
```suggestion
LOGGER = logging.getLogger(__name__)
```

Reply:
Resolving since I removed the logging altogether in favor of a warning as per your advice below.